openquake/hazardlib/gsim/sandikkaya_akkar_2017.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2023 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`SandikkayaAkkar2017Rjb`
+Module exports :class:`SandikkayaAkkar2017Repi`
+Module exports :class:`SandikkayaAkkar2017Rhyp`
+"""
+import numpy as np
+
+from openquake.baselib.general import CallableDict
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import CAV, IA
+
+
+_compute_logarithmic_distance_term = CallableDict()
+
```

Comment:
If I remember correctly, we don't use underscores for similar CallableDicts in other parts of hazardlib, I would just call it `compute_logarithmic_distance_term`.

Reply:
this worked. thanks!